src/main/kotlin/com/exactpro/th2/dataprocessor/zephyr/impl/standard/ZephyrEventProcessorImpl.kt

```diff
@@ -145,12 +145,12 @@ class ZephyrEventProcessorImpl(
             ExecutionUpdate(
                 id = execution.id,
                 status = executionStatus,
-                comment = "Updated by th2 because of event with id: ${event.eventId.id}"
+                comment = "Updated by th2 because of event with id: ${event.toJson()}"
```

Comment:
I don't think that we need the whole event in the JIRA comment. We probably need only an ID